adr/ADR-32.md

```diff
@@ -223,15 +246,44 @@ The type for this is `io.nats.micro.v1.schema_response`.
     * Average processing_time is the total processing_time divided by the num_requests
     */
     average_processing_time: Nanos;
-    /**
-    * ISO Date string when the service started in UTC timezone
-    */
-    started: string
 }
 ```
 
 The type for this is `io.nats.micro.v1.stats_response`.
 
+## Adding groups and endpoints
+
+A service can be extended by adding additional groups and endpoints.
+
+### Groups
+
+A group serves as a common prefix to all endpoints registered in it.
+A group can be created using `addGroup(name)` method on a Service.
+Group name should be a valid NATS subject, but cannot contain `>`
+wildcard (as group name serves as subject prefix).
+
```

Comment:
Group can also have an empty `""` name, reason for this is the process for the service can inject a prefix if necessary. Groups that are empty ignore the group name when building the subject. This allows for exporting a service to a particular account, etc.